EnglishFilter/sections/specific.txt

```diff
@@ -296,6 +296,8 @@ $third-party,script,domain=readcomiconline.li|filmyzilla.autos
 !
 !
 !
+wiki.gg##div[class^="sidebar-showcase"]
```

Comment:
Not sure is it is required on all subdomains. Looks like one game specific branding.
https://thoriummod.wiki.gg/wiki/Special:RecentChanges?hidebots=1&limit=50&days=7&enhanced=1&urlversion=2

Reply:
@Alex-302 Not one game specific, you can look at this too
https://darkdeity.wiki.gg/wiki/Weapons_and_Spells

Reply:
I also blocked these images and we need the rules above.